packages/ui/package.json

```
@@ -43,7 +43,7 @@
     "unocss": "^0.31.2"
   },
   "devDependencies": {
-    "nuxt3": "latest",
+    "nuxt": "^3.0.0-rc.0",
```

Let's use latest for internal repos :) 
```suggestion
    "nuxt": "npm:nuxt3@latest",
```